e2e/specs/patientSearch.spec.ts

```diff
@@ -9,7 +9,7 @@ test.beforeEach(async ({ api }) => {
   patient = await generateRandomPatient(api);
 });
 
-test('should be able to search patients by identifier', async ({ page, api }) => {
+test('should be able to search patients by identifier', async ({ page }) => {
```

Comment:
Good catch!

Reply:
eslint rule is not enforced. Found inside the esm modules files as well.